packages/ns-api/files/ns.devices

```diff
@@ -77,6 +77,24 @@ def list_devices():
 
     all_devices = merged_devices + ifaces_without_device
 
+    # retrieve hotspot interfaces
+
+    hotspot_config = utils.get_all_by_type(uci, 'dedalo', 'dedalo')
```

Comment:
The configuration is saved in a key named `config`
```suggestion
    hotspot_config = utils.get_all_by_type(uci, 'dedalo', 'config')
```

Reply:
Fixed. `try/except` is not needed since dedalo configuration is always present